core/src/components/datetime/test/showDaysOutsideCurrentMonth/datetime.e2e.ts

```diff
@@ -0,0 +1,48 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs().forEach(({ title, screenshot, config }) => {
+  test.describe(title('datetime: ShowDaysOutsideCustomMonth'), () => {
+    test('with custom-grid should not have any visual regressions', async ({ page }) => {
+      await page.goto('/src/components/datetime/test/showDaysOutsideCurrentMonth', config);
+      const customGrid = page.locator('#custom-grid');
+      await expect(customGrid).toHaveScreenshot(screenshot(`customGrid-with-hidden-days`));
```

Comment:
We should avoid using camelCase in file names for screenshots, prefix with the component name and what the test is for. 

```suggestion
      await expect(customGrid).toHaveScreenshot(screenshot(`datetime-show-adjacent-days-custom-grid`));
```